src/query/xasl_cache.c

```diff
@@ -399,13 +399,18 @@ xasl_cache_ent::init_clone_cache ()
 static void *
 xcache_entry_alloc (void)
 {
+  static int xcache_nentries = 0;
```

Comment:
I'm not sure it is safe

Reply:
it's called from xcache initialize routine at boot time.
so, the data is safe.